.github/workflows/latest-release-tag.yml

```diff
@@ -0,0 +1,95 @@
+name: Latest tag
+on:
+  release:
+    types: [published] # This makes it run only when a new released is published
+
+jobs:
+  latest-release:
+    name: Add/update tag to new release
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Check for latest tag
+      id: latest-tag
+      run: |
+        LATEST_TAG=$(git show-ref --tags -d | grep latest^{} | sed 's/refs\/tags\/latest^{}//') || echo 'not found'
```

Comment:
It took me some time to figure out what `^{}` means in this context (I thought it was some non-standard regex in grep 😆 ). Perhaps we could add a comment here explaining what we're greping and seding.

Reply:
I wound up changing the lookup on this, so that we didn't have to look for the tag with the pointer sha. It should read a bit better now.